pandora-client-web/src/components/wardrobe/wardrobeItems.tsx

```diff
@@ -98,8 +101,18 @@ export function useWardrobeItems(currentFocus: WardrobeFocus): {
 }
 
 export function WardrobeItemManipulation(): ReactElement {
-	const { target, targetSelector, assetList, heldItem, setHeldItem, focus } = useWardrobeContext();
+	const { target, targetSelector, assetList, heldItem, setHeldItem, focus, setScrollToItem } = useWardrobeContext();
+
+	const location = useLocation();
+	if (IsObject(location.state) && 'deviceId' in location.state) {
+		const focusItemId: ItemId = location.state.deviceId as ItemId;
+
+		focus.value = { container: [], itemId: focusItemId };
+		setScrollToItem(focusItemId);
+		location.state = {};
+	}
```

Comment:
I feel this should be inside a `useEffect`

Reply:
Done